src/Sfa.Tl.Matching.Web.UnitTests/Controllers/Provider/When_Provider_Controller_Index_Post_Is_Called.cs

```diff
@@ -31,7 +30,7 @@ public When_Provider_Controller_Index_Post_Is_Called()
 
             var logger = Substitute.For<ILogger<ProviderController>>();
 
-            var config = new MapperConfiguration(c => c.AddProfiles(typeof(SearchParametersViewModelMapper).Assembly));
```

Comment:
put this back to Assembly, its safe to load all the Mapping so that we can see any mapping related issues